src/colvarbias_opes.cpp

```diff
@@ -271,7 +284,11 @@ int colvarbias_opes::init(const std::string& conf) {
     key_lookup(conf, "grid", &grid_conf);
     m_reweight_grid.reset(new colvar_grid_scalar(m_pmf_cvs, nullptr, false, grid_conf));
     m_pmf_grid.reset(new colvar_grid_scalar(m_pmf_cvs, m_reweight_grid));
-    get_keyval(conf, "pmfHistoryFrequency", m_pmf_hist_freq, 0);
+    get_keyval(conf, "pmfHistoryFrequency", m_pmf_hist_freq, output_freq);
+    if ((m_pmf_hist_freq % output_freq) != 0) {
+      error_code |= cvm::error("Error: pmfHistoryFrequency must be a multiple of outputFreq.\n",
+                               COLVARS_INPUT_ERROR);
+    }
```

Comment:
@HanatoK Here, I wasn't completely sure between checking against `output_freq` or `time_step_factor`. We could go either way.